rspace/src/test/scala/coop/rchain/rspace/HistoryActionsTests.scala

```diff
@@ -182,6 +194,198 @@ trait HistoryActionsTests
 
       history.lookup(store.trieStore, channelsHash) shouldBe None
     }
+
+  "getCheckpoint, consume, reset" should "result in an empty store" in
+    withTestStore { store =>
+      val checkpoint0 = getCheckpoint(store)
+
+      val gnat1 = GNAT(List("ch1"),
+                       List.empty[Datum[String]],
+                       List(WaitingContinuation(List(Wildcard), new StringsCaptor, false)))
+
+      // val channelsHash1: Blake2b256Hash = store.hashChannels(gnat1.channels)
+
+      consume(store,
+              gnat1.channels,
+              gnat1.wks.head.patterns,
+              gnat1.wks.head.continuation,
+              gnat1.wks.head.persist)
```

Comment:
I'm not in favor of this.  

`consume`ing a `GNAT` doesn't really make sense as you are discarding the `data` part of the `GNAT`.  We are only using it here for the sake of having something to test against.

Because, `consume` and `produce` are the "core" functions in rspace,  I'm not terribly keen on overloading them in ways that don't really make sense just to save a few lines of code, even if just test helpers.

Reply:
yeah the suggestion was to clean up the test (as in have a helper in the test, not pollute the API)